Modules/posixmodule.c

```diff
@@ -5356,10 +5356,10 @@ os_posix_spawn_impl(PyObject *module, path_t *path, PyObject *argv,
     if (file_actions != Py_None) {
         /* There is a bug in old versions of glibc that makes some of the
          * helper functions for manipulating file actions not copy the provided
-         * buffers. The use of `temp_buffer` here is a workaround that keeps the
+         * buffers. The problem is that posix_spawn_file_actions_addopen copies
```

Comment:
Actually the problem is not that posix_spawn_file_actions_addopen copies the value of path, but that it isn't copies in on glibc <2.20.

Comments are plain text, no need to use the Markdown mark up (``` **path** ```, ``` `temp_buffer` ```).

Reply:
>Actually the problem is not that posix_spawn_file_actions_addopen copies the value >of path, but that it isn't copies in on glibc <2.20.
Sorry, bad redaction on my part. I was trying to express that `addopen` copies it except for older versions of glibc and the fact that these old versions don't copy it is the problem.


Fixed in c74b988